readme.md

```diff
@@ -16,7 +16,7 @@ Lighthouse requires Chrome 54 or later.
 
 Install from the Chrome Web Store: [chrome.google.com/webstore/detail/lighthouse/…](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk)
 
-Quick-start guide on using the Lighthouse extension: http://bit.ly/lighthouse-quickstart
+Quick-start guide on using the Lighthouse extension: <http://bit.ly/lighthouse-quickstart>
```

Comment:
`<>` means it's a link explicitly. Here it works because it's GitHub Markdown and they do it automatically :)